test/OpenTelemetry.Tests/Metrics/MetricViewTests.cs

```diff
@@ -292,6 +292,80 @@ public void ViewToSelectTagKeys()
             // Single point expected.
             Assert.Single(metricPoints);
         }
+
+        [Fact]
+        public void ViewToDropSingleInstrument()
+        {
+            using var meter = new Meter("ViewToDropSingleInstrumentTest");
+            var exportedItems = new List<Metric>();
+            using var meterProvider = Sdk.CreateMeterProviderBuilder()
+                .AddSource(meter.Name)
+                .AddView("counterNotInteresting", new MetricStreamConfiguration() { Aggregation = Aggregation.Drop })
+                .AddInMemoryExporter(exportedItems)
+                .Build();
+
+            // Expecting one metric stream.
+            var counterInteresting = meter.CreateCounter<long>("counterInteresting");
+            var counterNotInteresting = meter.CreateCounter<long>("counterNotInteresting");
+            counterInteresting.Add(10);
+            counterNotInteresting.Add(10);
+
+            meterProvider.ForceFlush(MaxTimeToAllowForFlush);
+            Assert.Single(exportedItems);
+            var metric = exportedItems[0];
+            Assert.Equal("counterInteresting", metric.Name);
+        }
+
+        [Fact]
+        public void ViewToDropMultipleInstruments()
+        {
+            using var meter = new Meter("ViewToDropMultipleInstrumentsTest");
+            var exportedItems = new List<Metric>();
+            using var meterProvider = Sdk.CreateMeterProviderBuilder()
+                .AddSource(meter.Name)
+                .AddView("server*", new MetricStreamConfiguration() { Aggregation = Aggregation.Drop })
+                .AddInMemoryExporter(exportedItems)
+                .Build();
+
+            // Expecting two client metric streams as both server* are dropped.
+            var serverRequests = meter.CreateCounter<long>("server.requests");
+            var serverExceptions = meter.CreateCounter<long>("server.exceptions");
+            var clientRequests = meter.CreateCounter<long>("client.requests");
+            var clientExceptions = meter.CreateCounter<long>("client.exceptions");
+            serverRequests.Add(10);
+            serverExceptions.Add(10);
+            clientRequests.Add(10);
+            clientExceptions.Add(10);
+
+            meterProvider.ForceFlush(MaxTimeToAllowForFlush);
+            Assert.Equal(2, exportedItems.Count);
+            Assert.Equal("client.requests", exportedItems[0].Name);
+            Assert.Equal("client.exceptions", exportedItems[1].Name);
+        }
+
+        [Fact]
+        public void ViewToDropAndRetainInstrument()
+        {
+            using var meter = new Meter("ViewToDropAndRetainInstrumentTest");
+            var exportedItems = new List<Metric>();
+            using var meterProvider = Sdk.CreateMeterProviderBuilder()
+                .AddSource(meter.Name)
+                .AddView("server.requests", new MetricStreamConfiguration() { Aggregation = Aggregation.Drop })
```

Comment:
Instead of having to create a new object, could we use something like `MetricStreamConfiguration.Drop`?

Reply:
Not very clean, but this commit attempts this : https://github.com/open-telemetry/opentelemetry-dotnet/pull/2449/commits/921449597a2aa03a51a9bf238057211a96ca125b
cc: @alanwest you raised this point in one of the early View PRs as well I think.

Reply:
Yea, I made the comment here https://github.com/open-telemetry/opentelemetry-dotnet/pull/2396#discussion_r717121867.

The somewhat wonky thing about this pattern is that a user can mutate the static instance. I see you protected the Aggregation property from being set by throwing an exception. If we create more static helpers like this we could consider overriding all of the properties and throw exceptions as necessary.